### DISCUSSION TITLE
Generated svg source in playground

### DISCUSSION THREAD

**Discussion by Sam-Apostel** [vercel/satori#229] [Ideas]
I noticed during testing I found myself wanting to check out the generated svg's source. 
I added a 'code' tab to the playground to view the generated svg's source. I think this might help some others debugging their jsx and css if they can see the direct changes reflected as code too.

There's a lot of potential for adding debugging functionality but I want to prevent rewriting the browser devtools.
How far would you like to see this go?

<img width="1680" alt="Screenshot 2022-10-17 at 19 33 52" src="https://user-images.githubusercontent.com/16917762/196245167-3462d782-3bc0-4c58-a23c-448bf7b4581b.png">

**Comment by shuding** [comment]
Love that idea!
